docs/examples/tls_mutual_authentication.rst

```diff
@@ -1,60 +1,45 @@
 TLS parameters example
-=============================
-This examples demonstrates a TLS session with RabbitMQ using mutual authentication.
+======================
 
-It was tested against RabbitMQ 3.6.10, using Python 3.6.1 and pre-release Pika `0.11.0`
-
-Note the use of `ssl_version=ssl.PROTOCOL_TLSv1`. The recent verions of RabbitMQ disable older versions of
-SSL due to security vulnerabilities.
-
-See https://www.rabbitmq.com/ssl.html for certificate creation and rabbitmq SSL configuration instructions.
+This examples demonstrates a TLS session with RabbitMQ using mutual authentication. It was tested against RabbitMQ 3.7.4, using Python 3.6.5 and Pika 1.0.0b1.
 
+See https://www.rabbitmq.com/ssl.html for certificate generation and RabbitMQ TLS configuration.
 
 tls_example.py::
 
-    import ssl
-    import pika
     import logging
+    import pika
+    import ssl
 
     logging.basicConfig(level=logging.INFO)
-
-    context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
-    context.verify_mode = ssl.CERT_REQUIRED
-    context.load_verify_locations('/Users/me/tls-gen/basic/testca/cacert.pem')
-    context.load_cert_chain('/Users/me/tls-gen/basic/client/cert.pem',
-                            '/Users/me/tls-gen/basic/client/key.pem')
-
-    cp = pika.ConnectionParameters(ssl_options=pika.SSLOptions(context))
-
-    conn = pika.BlockingConnection(cp)
-    ch = conn.channel()
-    print(ch.queue_declare("sslq"))
-    ch.publish("", "sslq", "abc")
-    print(ch.basic_get("sslq"))
-
+    context = ssl.create_default_context(
+        cafile="/Users/me/tls-gen/basic/testca/cacert.pem")
+    context.load_cert_chain("/Users/me/tls-gen/basic/client/cert.pem",
+                            "/Users/me/tls-gen/basic/client/key.pem")
+    server_hostname = "example.com"
```

Comment:
Please apply the suggested changes in this function as well: https://github.com/pika/pika/blob/9fb234c0e91b709b5935f4d97432c5991597c61f/tests/acceptance/async_test_base.py#L120